Can't found tools.jar when building RelEx

ant
Unable to locate tools.jar. Expected to find it in /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
Buildfile: /tmp/link-grammar-5.3.7/build/bindings/java/build.xml

build-subprojects:

init:
    [mkdir] Created dir: /tmp/link-grammar-5.3.7/build/bindings/java/bin
     [copy] Copying 5 files to /tmp/link-grammar-5.3.7/build/bindings/java/bin

build-project:
     [echo] linkgrammar: /tmp/link-grammar-5.3.7/build/bindings/java/build.xml

BUILD FAILED

--- a/scripts/hrtool
+++ b/scripts/hrtool
@@ -64,6 +64,7 @@ install_relex_deps() {
     sudo apt-get -y update
     sudo apt-get -y install build-essential python-dev swig zlib1g-dev unzip wget
     sudo apt-get -y install wordnet-dev wordnet-sense-index
+    sudo apt-get -y install openjdk-7-jdk
     sudo apt-get -y install ant libcommons-logging-java libgetopt-java
 
     wget http://raw.github.com/opencog/ocpkg/master/ocpkg -O octool

